components/playground/playground.go

```diff
@@ -924,6 +859,17 @@ func (p *Playground) bootCluster(options *bootOptions) error {
 			if grafana != nil {
 				_ = syscall.Kill(grafana.cmd.Process.Pid, sig)
 			}
+		} else {
```

Comment:
why SIGKILL? that means always SIGKILL when user press ctrl-c ?

can we gracefully stop it first, and use SIGKILL if user press again ctrl-c again or some reach some timeout.